app/views/layouts/application.html.erb

```diff
@@ -20,6 +20,11 @@
 
     <title><%= @page_title + ' - ' if !@page_title.nil? %><%= PAGE_TITLE_SUFFIX %></title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <script type="text/javascript">
+      window.OX = window.OX || {}
+      OX.I18n = <%= (t :javascript).to_json.html_safe %>
```

Comment:
Yes, this is a neat way to do lightweight client-side translations without having to pull in an entire JS i18n lib.